src/core/electric_load.jl

```diff
@@ -158,19 +158,13 @@ mutable struct ElectricLoad  # mutable to adjust (critical_)loads_kw based off o
             end
```

Comment:
Adding a note here to require the year if a custom load profile is updated (here and for the heating loads)

Reply:
Added this requirement. Validation to check if year is not nothing can happen above the check for loads_kw, doe_reference_name, etc, since the default is assigned for CRB without a year input.

Reply:
@Bill-Becker I think it might be worth adding this to the CHANGELOG too

Reply:
Added to changelog.